contributing/documentation/standards.rst

```diff
@@ -175,6 +175,27 @@ In addition, documentation follows these rules:
   * his or hers, use theirs
   * himself or herself, use themselves
 
+Moreover, we try to avoid belittling words as much as possible. Usually, the people
```

Comment:
What about including this as a list item like above? (also took the freedom to suggest a slightly shorter way of explaining this. The shorter a "guideline", the more likely someone will follow it)

```rst
* **Punctuation**: ...
* **Pronouns**: ...
* **Gender-neutral language**: ...

   * he or she, use they
   * ...
* **Avoid belittling words**: People read documentation because they know very little about
   a specific topic or are even completely new to it. Things that seem "obvious" or "simple"
   for the documentor, can be the exact opposite for the reader. To make sure everybody
   feels comfortable when reading the documentation, try to avoid words like:

   * ...
```

Reply:
Nice proposal! Done in https://github.com/symfony/symfony-docs/pull/11685/commits/8a6daf4b276197d1324482349fb421596886124c. I've replaced "documentor" though, as I'm not sure this is an existing English word 😄